package.json

```diff
@@ -88,7 +89,7 @@
     "style-loader": "0.13.1",
     "url-loader": "0.5.7",
     "webpack": "1.13.2",
-    "webpack-bundle-analyzer": "1.4.1",
+    "webpack-bundle-analyzer": "^1.5.3",
```

Comment:
Can you please add a line to this table to document this script? https://github.com/coryhouse/react-slingshot/blob/master/docs/FAQ.md#what-do-the-scripts-in-packagejson-do